lib/services/api/messages.ts

```diff
@@ -11,6 +11,11 @@ export const API_MESSAGE_CHANGE_PW_IN_PROGRESS     = 'An existing change passwor
 
 export const API_MESSAGE_FALLBACK_LOGIN_FAIL       = 'Invalid email or password.';
 
+export const API_MESSAGE_GENERIC_TOKEN_REFRESH_FAIL= `A server error occurred while trying to refresh your session.
+                                                      Please try again.`;
+
+export const API_MESSAGE_TOKEN_REFRESH_IN_PROGRESS = 'The session is being updated with new tokens, please standby.';
```

Comment:
"Your account session is being renewed with the server. Please try your request again."